coupa/countries
# Update IN.yaml for JZ-51601

- [x] @davidejorgensen 
- [ ] @mekhovov  
- [x] @luckyx1  

Copied the latest In file from the countries gem 

Changes made:

1.  "Daman and Diu" and "Dadra and Nagar" is now "Dadra and Nagar Haveli and Daman and Diu" (ISO code for the new entry is IN-DH)
2. Add "Ladakh" (ISO code is IN-LA)
3. Remove the EXTRA "Telangana" (keep one of them, ISO code to keep should be IN-TG)
4. Remove "Uttaranchal" (leave "Uttarakhand", ISO code to keep should be IN-UT)
